index.js

```diff
@@ -247,6 +250,30 @@ class Analytics {
         done(err)
       })
   }
+
+  _retryCondition (error) {
```

Comment:
Might rename this `shouldRetry` or `isErrorRetryable` for clarity.

Reply:
Renamed to `isErrorRetryable`

Reply:
I think you might have forgotten to push?

Reply:
Oops! Pushed now.